src/styles/product.scss

```diff
@@ -0,0 +1,93 @@
+:root {
+  margin: 0;
```

Comment:
It's better to apply these styles directly to the body element instead of :root.